rosdep/base.yaml

```
@@ -6950,6 +6950,16 @@ pcre-dev:
   nixos: [pcre]
   rhel: [pcre-devel]
   ubuntu: [libpcre++-dev]
+pcscd:
+  alpine: [pcsc-lite]
+  arch: [pcsclite]
+  debian:
+    bookworm: [pcsc-lite]
```

```suggestion
    bookworm: [pcsc-lite]
```

I can't seem to find this package in https://packages.debian.org.  Am I missing something?  Shouldn't this just be `pcscd`?